models/mat2NV.jl

```diff
@@ -0,0 +1,34 @@
+using NeuralVerification, MAT
+using NeuralVerification: Layer, Network, ReLU, Id
+
+_vec(A::AbstractMatrix) = vec(A)
+_vec(A::Number) = [A]
+_vec(A::AbstractVector) = A
+
+function readnn(file)
+    vars = matread(file)
+    if(first(keys(vars)) == "controller")
```

Comment:
there may be other keys. i would suggest to do:

```
!haskey(vars) && throw(ArgumentError("didn't find key \"controller\"")
```

Reply:
in fact, let't make "controller" a keyword argument with default value, so we don't hard-code it here.